RoddyCore/src/de/dkfz/roddy/knowledge/files/FileGroup.java

```diff
@@ -32,6 +35,21 @@ public FileGroup(List<F> files) {
         }
     }
 
+    @Override
+    public Iterator<F> iterator() {
+        return filesInGroup.iterator();
+    }
+
+    @Override
+    public void forEach(Consumer<? super F> action) {
```

Comment:
Interesting that this one also has to be overridden despite the iterator() override.

Reply:
I actually don't know. I think this overrides the Java .forEach method whereas iterator is the one used by "for in" or Groovy .each